src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
Create new Random() class-level variable, don't need to create it each time when the method getRandomColor() is called